### ISSUE TITLE
Packer Fails When Network is Public for `hypervisor-iso` 

### ISSUE THREAD

**Issue by shogerr** [StefanScherer/packer-windows#185]
I encountered this issue while building Windows 10 with WinRM connectivity.

After finishing all of the updates provisioned in `win-updates.ps1`, a call to `enable-winrm.ps1` is made.

The packer build will hang with "Waiting for WinRM to become available..".

WinRM is never being enabled because the `enable-winrm.ps1` script will fail if the current network is set to public. Specifically, calls to `winrm quickconfig` will fail with an error message.

If the following instructions are put at the top of `enable-winrm.ps1`, then the WinRM connection will succeed with every build.

```
$networkListManager = [Activator]::CreateInstance([Type]::GetTypeFromCLSID([Guid]"{DCB00C01-570F-4A9B-8D69-199FDBA5723B}")) 
$connections = $networkListManager.GetNetworkConnections() 

# Set network location to Private for all networks 
$connections | % {$_.GetNetwork().SetCategory(1)}
```
The above is taken from http://www.hurryupandwait.io/blog/fixing-winrm-firewall-exception-rule-not-working-when-internet-connection-type-is-set-to-public.

This behavior does not occur every time. Out of four builds, I have had WinRM work out-of-the-box one time. All other build events were a failure until the network was set to public in the running VM.

**Comment by StefanScherer** [comment]
Thanks. When you have a fix already, do you want to send a pull request? Thanks 😀

**Comment by shogerr** [comment]
Sounds good Stefan. I am going to be working with Hyper-V a bit more, so I may have a PR up soon.

**Comment by StefanScherer** [comment]
Fixed with #195
